mandatory/2-function-creation.js

```diff
@@ -4,48 +4,103 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+let userAge = 19;
+let isLoggedIn = true;
+function isAcceptableUser(userAge, isLoggedIn){
+  if (userAge >= 18 && isLoggedIn === true){
+  return true;
+ } else {
+  return false;
+ }
+}
 
 /*
   Complete the function to apply discount percent based on how much is totalPrice in user cart.
     - When the total price is greater than 200, a 10% discount should be applied
     - When the total price is less than 200, a 5% discount should be applied
 
   The function should return the new price to be paid (e.g. if the totalPrice is 150, a 5% discount
-    is applieds and 142.5 should be returned)
+    is applied and 142.5 should be returned)
 */
-
-function applyDiscount(totalPrice) {}
-
+//let totalPrice = 150;
+function applyDiscount(totalPrice) {
+  if (totalPrice >200){
+    return (totalPrice - totalPrice * 0.01);
```

Comment:
nice job with the if statement! 

Just might want to check your maths on this one 😉
0.01 would be 1% and 0.1 would be a 10%, but where is our 5% discount? 😜

Reply:
Thanks. I understand 😅